packages/preact/src/index.ts

```diff
@@ -323,6 +323,25 @@ export function useComputed<T>(compute: () => T) {
 	return useMemo(() => computed<T>(() => $compute.current()), []);
 }
 
+export function useSignalEffect(cb: () => void | (() => void)) {
```

Comment:
@JoviDeCroock is the param type duplicated?

Reply:
no you can return void or a function that returns void i.e. a clenaup function